### PR TITLE
Javascript/javascript1/week2

### PR DESCRIPTION
**Pull request by Enqira**

### CODE REVIEW

File: javascript/javascript1/week2/eventApp.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+var weekDays = ["Sunday","Monday","tuesday","Wednesday ","Thursday","Friday","Saturday"];
+
+function getEventWeekday(daysLeft) {
+    
+    var eventDate = new Date()
```
Comment (aina21) [inline javascript/javascript1/week2/eventApp.js:6]: dont use var

File: javascript/javascript1/week2/studentManager.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+const class07Students = ["mo", "mmm"];
+function addStudentToClass(studentName) {
+    let countedNumberOfStudents = getNumberOfStudents()
```
Comment (aina21) [inline javascript/javascript1/week2/studentManager.js:3]: const

File: javascript/javascript1/week2/studentManager.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    } 
+    
+    else if ( studentName == "Queen"){
+        class07Students.push(studentName)
+        return
+    }
+    else if ( class07Students.includes(studentName)){
+        return "already included"
+    }
+    else if (countedNumberOfStudents > 5){
```
Comment (aina21) [inline javascript/javascript1/week2/studentManager.js:15]: Do you mean >6

File: javascript/javascript1/week2/studentManager.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+const class07Students = ["mo", "mmm"];
+function addStudentToClass(studentName) {
+    let countedNumberOfStudents = getNumberOfStudents()
+    if (studentName == ""){
+        return "empty array"
+    } 
+    
+    else if ( studentName == "Queen"){
```
Comment (aina21) [inline javascript/javascript1/week2/studentManager.js:8]: what if we add queen twice?

**Review by aina21 — CHANGES_REQUESTED** [review]
good job 👍

File: javascript/javascript1/week2/eventApp.js (hunk 1 above)
Comment (Enqira) [reply javascript/javascript1/week2/eventApp.js:6]: Ok, Thanks

File: javascript/javascript1/week2/studentManager.js (hunk 3 above)
Comment (Enqira) [reply javascript/javascript1/week2/studentManager.js]: >5 works fine, but for better readibility >=6 should be used

File: javascript/javascript1/week2/studentManager.js (hunk 4 above)
Comment (Enqira) [reply javascript/javascript1/week2/studentManager.js]: Fixed that isue, Thanks

**Pull request merged**